package.json

```diff
@@ -40,6 +40,8 @@
     }
   },
   "dependencies": {
-    "jest": "^26.6.3"
+    "has-flag": "^4.0.0",
```

Comment:
I wasn't sure whether I need to add dependencies in this package.json file, or whether adding them to the cli folder's package.json is sufficient. I would love to discuss this in more detail in our 1:1 tomorrow, so I can have a better understanding of this in the future.

Reply:
I don't think we need to add this as a dep. Ideally we can reduce the overall number of deps in the project. This dep is a single small function that we could easily wrap up into `supportHyperlinks`. https://github.com/sindresorhus/has-flag/blob/main/index.js